README.md

```diff
@@ -1,2 +1,62 @@
-# OpenCL-CTS [![Build Status](https://api.travis-ci.org/KhronosGroup/OpenCL-CTS.svg?branch=master)](https://travis-ci.org/KhronosGroup/OpenCL-CTS/branches)
-The OpenCL Conformance Tests
+OpenCL-CTS [![Build Status](https://api.travis-ci.org/KhronosGroup/OpenCL-CTS.svg?branch=master)](https://travis-ci.org/KhronosGroup/OpenCL-CTS/branches)
+=================
+
+This document describes how to build the OpenCL Conformance Test suite.
+
+Requirements
+------------
+
+### Common
+The following tools must be installed and present in the PATH variable:
+
+ * Git (for checking out sources)
+ * CMake 3.5.1 or newer
+
+### Win32
+Visual Studio 2015 or newe
+
+### Linux
+Standard toolchain (make, gcc/clang)
+
+### Android
+
+ * Android NDK r17c or later.
+ * Android SDK with: SDK Tools, SDK Platform-tools, SDK Build-tools, and API 28
+
+If you have downloaded the Android SDK command line tools package (25.2.3 or higher) then you can install the necessary components by running:
+
+	tools/bin/sdkmanager tools platform-tools 'build-tools;25.0.2' 'platforms;android-28'
+
+Building CTS
+------------
+With CMake, out-of-source builds are always recommended. Create a build directory
+of your choosing, and in that directory generate Makefiles or IDE project
+using cmake.
```

Comment:
Consider saying that the command lines below should be executed from the top-level directory with the CTS source?